library/gazebo

```diff
@@ -75,12 +75,30 @@ Directory: gazebo/7/ubuntu/xenial/libgazebo7
 ########################################
 # Distro: ubuntu:xenial
 
-Tags: gzserver8
+Tags: gzserver8, gzserver8-xenial
 Architectures: amd64
 GitCommit: 413df81c5f25236563fda9505c89a84f18448132
 Directory: gazebo/8/ubuntu/xenial/gzserver8
 
-Tags: libgazebo8, latest
+Tags: libgazebo8, libgazebo8-xenial
 Architectures: amd64
 GitCommit: 413df81c5f25236563fda9505c89a84f18448132
 Directory: gazebo/8/ubuntu/xenial/libgazebo8
+
+
+################################################################################
+# Release: 9
+
+########################################
+# Distro: ubuntu:xenial
+
+Tags: gzserver9, gzserver9-xenial
+Architectures: amd64
+GitCommit: b8deb76898b59e0d43518f2cbdca6c90afc1debe
+Directory: gazebo/9/ubuntu/xenial/gzserver8
+
+Tags: libgazebo9, libgazebo9-xenial, latest
+Architectures: amd64
+GitCommit: b8deb76898b59e0d43518f2cbdca6c90afc1debe
+Directory: gazebo/9/ubuntu/xenial/libgazebo8
```

Comment:
Looks like it is because these new folders are still "8" (and so it can't scrape a `FROM` out of the Dockerfile).